### PR TITLE
Task authors

### PR DESCRIPTION
**Pull request by atd**

### CODE REVIEW

File: api.raml [hunk 1]
```diff
@@ -33,6 +33,12 @@ baseUri: http://api.circular.works
                             "id": "25"
                           }
                         ]
+                      },
+                      "author": {
+                        "data": {
+                          "type": "users",
+                          "id": "3"
+                        }
```
Comment (nogates) [inline api.raml]: Do we need the author populated on the index endpoint? Looks like the kind of information that should be loaded when the user is visiting the `show#task` endpoint

File: app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -18,6 +18,8 @@ def show
   def create
     @task = Task.new(task_params)
 
+    @task.author = current_user
```
Comment (nogates) [inline app/controllers/tasks_controller.rb]: not sure if I prefer `current_user.tasks.build(task_params)`... one line is better? :P

**Review by nogates — APPROVED** [review]
Awesome! just a quick question about not putting too much stuff on the `task#index` endpoint

File: app/controllers/tasks_controller.rb (hunk 2 above)
Comment (atd) [reply app/controllers/tasks_controller.rb]: I was used to it before params white lists were implemented :innocent: 

I am changing it

File: api.raml (hunk 1 above)
Comment (atd) [reply api.raml]: Probably not. We have to check how to serialize different data in different points though.

**Pull request merged**